Lib/fontTools/feaLib/builder.py

```diff
@@ -89,6 +90,25 @@ def addOpenTypeFeaturesFromString(
     addOpenTypeFeatures(font, featurefile, tables=tables, debug=debug)
 
 
+class EmptyGposLookup(LookupBuilder):
+    """A stand-in for when you want to insert empty lookups for varLib to
+    combine with the actual ones later."""
+
+    def __init__(self, font, location):
+        super().__init__(font, location, "GPOS", 2)
+
+    def build(self):
+        from fontTools.ttLib.tables import otTables as ot
+
+        lookup = ot.Lookup()
+        lookup.LookupType = 2
+        lookup.LookupFlag = 0
+        lookup.SubTable = []
```

Comment:
Are we sure that we don't actually want tp have `lookup.SubTable = None` here instead of the empty list?
We need to check what varLib makes of these, whether the it disinguishes one with SubTable=[] vs one with SubTable=None. In general what we want in these cases is that this particular lookup "does not participate" in variations basically.

Reply:
This would be really interesting indeed, to allow sparse models at the lookup level.

Reply:
but it's impossible with FEA because truly empty lookups have no type, it's their rules that give them their type. Unless you want to hack this by adding dummy `pos .notdef .notdef 0` ...

Reply:
It's okay to give the lookup type 0. Except that both fonttools compiler and varLib would choke on it then. :(

Reply:
Can we extend the format?

Reply:
we could...